src/unix/proctitle.c

```diff
@@ -84,10 +94,15 @@ int uv_set_process_title(const char* title) {
   if (process_title.len == 0)
     return 0;
 
+  uv_once(&once, init_once);
+  uv_mutex_lock(&mutex);
```

Comment:
This needs to be moved to before the `process_title.len == 0` check.  I'd change it to this:
```c
uv_once(&once, init_once);
uv_mutex_lock(&mutex);
if (process_title.len != 0) {
  // ...
}
uv_mutex_unlock(&mutex);
return 0;
```